proc_CPMG.py

```diff
@@ -3,29 +3,77 @@
 from proc_scripts import *
 from proc_scripts import postproc_dict
 from proc_scripts.fitting import decay
+from sympy import symbols
+import sympy as sympy
```

Comment:
maybe address w/ @aabeaton -- this has a conditional "if spincore" --> but that's exactly the purpose of having preprocessing.

Reply:
you don't need to address w/ @aabeaton -- see my other comments first, which I think will resolve this issue

Reply:
yes, but rather than specifying this explicitly, you can check to see if there are 1 or 2 phase cycling dimensions, whether or not there is an nScans dimension, and then respond accordingly